.github/actions/config-inversion-check/action.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Config Inversion Diff Check'
```

Comment:
Is somewhere explained what "Config Inversion" means? I've seen this term around the PR and it is not immediately obvious to me what it means.

If it is meant to be descriptive for devs in this repo, maybe we should use a more familiar or descriptive term about what it is? (e.g. Config Validation)